design1.css

```diff
@@ -0,0 +1,25 @@
+*{
+    margin: 0;
+    padding: 0;
+    height: 100%;
```

Comment:
Height 100% SHOULD NOT be a global property. Assign it to just `html` or `body` tags